UnitTests/UnitTests.FluentSharp.CoreLib/APIs/Test_API_NuGet.cs.orig

```diff
@@ -0,0 +1,185 @@
+<<<<<<< HEAD
+﻿using FluentSharp.CoreLib;
```

Comment:
same as above

Note: if there are others below like this, I will not comment on them anymore (since you get the idea)